jwst/datamodels/schemas/image.schema.yaml

```diff
@@ -31,6 +31,11 @@ allOf:
       fits_hdu: AREA
       default: 0.0
       datatype: float32
+    pathloss_pointsource:
+      title: Pathloss correction
+      fits_hdu: PS
```

Comment:
The corresponding HDU names for the NIRSpec pathloss corrections were much longer and more complicated. I like the short "PS", but that's now inconsistent with the equivalent NIRSpec point-source correction name.

Reply:
True.  I should rename the fits_hdu to pathloss_pointsource.